include/eve/detail/abi.hpp

```diff
@@ -25,9 +25,7 @@
 
 // Force a lambda function to be inline
 #if !defined(EVE_LAMBDA_FORCEINLINE)
-#  if defined(_MSC_VER)
-#    define EVE_LAMBDA_FORCEINLINE __forceinline
-#  elif defined(__GNUC__) && __GNUC__ > 3
+#  if defined(__GNUC__) && __GNUC__ > 3
```

Comment:
This is used in one place. We should delete it.

Reply:
Done